velox/common/caching/AsyncDataCache.h

```diff
@@ -667,9 +669,34 @@ class CacheShard {
 
 class AsyncDataCache : public memory::Cache {
  public:
+  struct Config {
+    Config(
+        double _maxWriteRatio = 0.7,
+        double _ssdSaveIntervalRatio = 0.125,
+        int32_t _minSavePages = 4096 /*16MB*/)
+        : maxWriteRatio(_maxWriteRatio),
+          ssdSaveIntervalRatio(_ssdSaveIntervalRatio),
+          minSavePages(_minSavePages){};
+
+    /// Maximum ratio of adding entries to a write batch. If SSD save
+    /// is slower than storage read, we must not have a situation where SSD save
+    /// pins everything and stops reading, therefore, we should stop adding more
+    /// entries when it exceeds this threshold.
+    const double maxWriteRatio;
+
+    /// SSD save when the number of pages that the new entries have has exceeded
+    /// (currentPages * ssdSaveIntervalRatio).
+    const double ssdSaveIntervalRatio;
+
+    /// Minimum number of pages the new entries should have to be able to
+    /// trigger SSD save.
+    const int32_t minSavePages;
```

Comment:
```
/// The min ratio of SSD savable (in-memory) cache space over the total cache space. Once the ratio exceeds this limit, we start writing SSD savable cache entries into SSD cache.
const double ssdSavableRatio;
/// Min SSD savable (in-memory) cache space to start writing SSD savable cache entries into SSD cache.
///
/// NOTE: we only write to SSD cache when both above conditions satisfy.
const int32_t minSsdSavableBytes;
```
Let's convert to minSsdSavableBytes to pages internally?  Bytes is more user-friend unit. Thanks!